serverless/aws/lambda/src/main/scala/sttp/tapir/serverless/aws/lambda/AwsServerInterpreter.scala

```diff
@@ -55,19 +51,17 @@ trait AwsCatsEffectServerInterpreter[F[_]] {
   }
 }
 
-object AwsCatsEffectServerInterpreter {
+object AwsServerInterpreter {
```

Comment:
I agree that it would be nice to have a single interpreter for all effect types, however I'd like to keep `MonadError` usage internal, so that users don't have to be bothered with yet another monad concept. From a user perspective, it's easier to just use a dedicated Future/cats-effect interpreter, instead of having to provide yet another implicit. We can share code between the two internally of course, but the user-facing interface should be specialised